README.md

```diff
@@ -59,3 +60,14 @@
 - [neo-gui](https://github.com/neo-project/neo-gui/) - Official wallet built by NEO team, which has a UI.
 - [neon-wallet](https://github.com/CityOfZion/neon-wallet) - Light wallet for the NEO blockchain.
 - [neotracker.io](https://neotracker.io/wallet) - NEO web wallet.
+
+## Information blogs
+
+- [The NEO Subreddit](https://www.reddit.com/r/NEO/).
+- [Neo News Today](https://neonewstoday.com/) - Community-made website with news and information, including Q&A's.
+- [Da Hongfei Twitter account](https://twitter.com/dahongfei) - Official Twitter account of Neo founder Da Hongfei.
+- [Erik Zhang Twitter account](https://twitter.com/neoerikzhang) - Official Twitter account of Neo founder Erik Zhang.
+
+## dApps
+
+- [A list of dApps for Neo](https://github.com/jhepkema/awesome-neo/blob/master/dApps.md) - Curated list of Neo dApps, in multiple stages of development.
```

Comment:
Change from `A list of dApps for Neo` to `List of dApps`.

And move this section up the page so that the section headings are in A-Z order.